cmd/core/main.go

```diff
@@ -131,6 +131,8 @@ func main() {
 	flag.DurationVar(&retryPeriod, "leader-election-retry-period", 2*time.Second,
 		"The duration the LeaderElector clients should wait between tries of actions")
 	flag.BoolVar(&enableClusterGateway, "enable-cluster-gateway", false, "Enable cluster-gateway to use multicluster, disabled by default.")
+	flag.StringVar(&ctrlClient.CachedGVKs, "cached-gvks", "", "Types of resources to be cached. For example, --cached-resources=Deployment.v1.apps,Job.v1.batch")
```

Comment:
all optimization args should have a common prefix, lets name it "optimize-cache-gvks"

Reply:
Solved.